cmd/go-mockgen/args.go

```diff
@@ -126,6 +131,10 @@ func parseManifest() ([]*generation.Options, error) {
 			opts.ForTest = true
 		}
 
+		if len(opts.Paths) > 0 && len(opts.Archives) > 0 {
+			return nil, fmt.Errorf("multiple import paths and archives are mutually exclusive")
```

Comment:
```suggestion
			return nil, fmt.Errorf("import paths and archives are mutually exclusive")
```

Is this more correct? They're both >0, not >1.